_docs/tasks/telemetry/querying-metrics.md

```diff
@@ -24,15 +23,7 @@ application.
 
 ## Querying Istio Metrics
 
-1. To query the metrics provided by Mixer, first install the Prometheus add-on.
-
-   In Kubernetes environments, execute the following command:
-
-   ```bash
-   kubectl apply -f install/kubernetes/addons/prometheus.yaml
-   ```
-
-1. Verify that the service is running in your cluster.
+1. Verify that the prometheus service is running in your cluster.
```

Comment:
add 
Verify that the prometheus service is running in your cluster (since 0.8 the prometheus setup is included in istio.yaml and istio-auth.yaml by default)